core/src/lib.rs

```diff
@@ -10,5 +10,7 @@
 
 extern crate alloc;
 
+pub mod page;
+pub mod pageId;
```

Comment:
recommend combining the page / page ID into one `page` module. if not, this should be `page_id` - Rust is always `snake_case`

Reply:
Oh, yes, I'm stupid 😅